techniques/general/G213.html

```diff
@@ -101,33 +97,33 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 
 		<section id="description">
 
-			<h2>Description</h2>
+			<h2>解説</h2>
 	
-			<p>The objective of this technique is to ensure that:</p>
+			<p>この達成方法の目的は、次を保証することである:</p>
 			<ol>
-				<li>People who use mounted devices or who cannot precisely manipulate a device are able to provide inputs through more conventional user interface components, and</li>
-				<li>People who may accidentally activate sensors due to tremors or other motor impairments have the ability to turn off motion actuation to prevent such accidental triggering of functions.</li>
+				<li>何かに取り付けられたデバイスを使用する人、又はデバイスを正確に操作できない人は、従来のユーザインタフェース コンポーネントを通じて、様々な入力を提供することができること、及び、</li>
+				<li>振戦、又はその他の運動障害のために誤ってセンサーを作動させてしまう可能性のある人は、意図せず機能を動作させることを防ぐため、動きによる起動をオフにすることができること。</li>
 			</ol>
 			
-			<p>When a device sensor such as an accelerometer or gyroscope is used to gather input:</p>
+			<p>加速度計、ジャイロスコープなどのデバイスセンサーを使用した入力を収集する場合:</p>
 			<ol>
-				<li>A button, link, or other conventional control is provided that does not rely on sensor input, and</li>
-				<li>A setting is provided to allow the user to turn off the sensor data.</li>
+				<li>センサー入力に依存しない、ボタン、リンク、又は従来のコントロールが提供されること、及び、</li>
+				<li>センサーのデータをユーザーがオフにできる設定を提供すること。</li>
 			</ol>			
 		
 		</section>
 
 		<section id="examples">
-			<h2>Examples</h2>
+			<h2>事例</h2>
 			<section class="example" id="example-1-shake-to-undo">
-				<h3>Example 1: Shake to undo</h3>
-				<p>After text is entered in a field, shaking a device shows a dialog offering users to undo the entry. Supporting use of the backspace key and/or providing a clear button next to the text field offers the same functionality.</p>
-				<p>Shake to undo can be turned off in a settings page.</p>
+				<h3>事例 1: 元に戻すためにシェイクする</h3>
+				<p>フィールドにテキストを入力した後、デバイスを振ると、入力を元に戻すためのダイアログが表示される。バックスペースキーの使用をサポートすること、及び／又はテキストフィールドの横にクリアボタンを提供することで、同じ機能を表す。</p>
+				<p>元に戻すためにシェイクする機能は、設定の画面でオフにすることができる。</p>
 			</section>
 			<section class="example" id="example-2-motion-activated-slider">
-				<h3>Example 2: Motion Activated Slider</h3>
-				<p>A slider can be adjusted by tipping the device to the left and right. There are also buttons to achieve the same functionality, and a tick-box that prevents the motion from having an effect.</p>
-				<p>Working example of a <a href="../../working-examples/device-motion-sensor-input/">slider with motion actuation</a>.</p>
+				<h3>事例 2: 動きで操作するスライダー</h3>
+				<p>スライダーは、デバイスを左右に傾けることで調整できる。同じ機能を実現するためのボタン、及び動きが効果を発揮しないようにするチェックボックスもある。</p>
+				<p><a href="../../working-examples/device-motion-sensor-input/">動きで操作するスライダー</a>の動作例。</p>
```

Comment:
リンク先が相対パスになっているので修正しました。
```suggestion
				<p><a href="https://www.w3.org/WAI/WCAG22/working-examples/device-motion-sensor-input/">動きで操作するスライダー</a>の動作例。</p>
```